packages/express/src/types.ts

```diff
@@ -0,0 +1,11 @@
+import { LogtoConfig } from '@logto/node';
+
+declare module 'http' {
+  interface IncomingMessage {
+    session: Record<string, string | undefined>;
+  }
+}
+
+export type LogtoExpressConfig = LogtoConfig & {
+  baseUrl: string;
```

Comment:
Can we add `baseUrl` to `LogtoConfig` in node sdk? It seems it is also used in next.js sdk.

Reply:
But it doesn't make sense for Node SDK, because `baseUrl` is useless in node sdk.

Reply:
OK